source/includes/iam-user-policy-minimum.json

```
@@ -0,0 +1,10 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": ["kms:Decrypt", "kms:Encrypt"],
+      "Resource": "<the Amazon resource name (ARN) of your symmetric key>"
```

Issue: I think we refer to it as "remote master key" rather than the more general descriptive term "symmetric key". Since there's no connection established between the two terms, there's a better chance the reader will recognize the one that was introduced.
Also, I think ARN is a proper noun(see https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html), especially when used before an all-caps abbreviation.

```suggestion
      "Resource": "<the Amazon Resource Name (ARN) of your remote master key>"
```